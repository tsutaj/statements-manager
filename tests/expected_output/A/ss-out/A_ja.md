


## Problem A

### <u>**A+B**</u>

2 つの整数 $A$, $B$ が与えられます。$A + B$ の計算結果を出力してください。

### 入力

入力は以下の形式で標準入力から与えられます。

```
$A$ $B$
```

### 出力

$A + B$ の計算結果を標準出力に出力してください。

### 制約

- $1 \leq A, B \leq 10^{9}$



### 入力例 1
```
3 5
```



### 出力例 1
```
8
```





### 入力例 2
```
10000 100
```



### 出力例 2
```
10100
```








