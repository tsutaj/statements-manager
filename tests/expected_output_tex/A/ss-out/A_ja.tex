\documentclass[11pt,a4paper]{article}
\usepackage{amsmath,amssymb}
\usepackage{ascmac}

\begin{document}


\section{Problem A}

\subsection{<u>**A+B**</u>}

2 つの整数 $A$, $B$ が与えられます。$A + B$ の計算結果を出力してください。

\subsubsection{入力}

入力は以下の形式で標準入力から与えられます。

\begin{screen}
$A$ $B$
\end{screen}

\subsubsection{出力}

$A + B$ の計算結果を標準出力に出力してください。

\subsubsection{制約}

\begin{itemize}
    \item $1 \leq A, B \leq 10^{9}$
\end{itemize}



\subsection{ 入出力例 }



\begin{itembox}[l]{ 入力例 1 }
3 5

\end{itembox}



\begin{itembox}[l]{ 出力例 1 }
8

\end{itembox}







\begin{itembox}[l]{ 入力例 2 }
10000 100

\end{itembox}



\begin{itembox}[l]{ 出力例 2 }
10100

\end{itembox}







\end{document}