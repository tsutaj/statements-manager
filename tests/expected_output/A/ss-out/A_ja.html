<!DOCTYPE html>
<html lang="ja">

<head>
    <title>Problem Statement</title>
    <meta charset="utf-8">
    <link href='http://fonts.googleapis.com/css?family=Noto+Serif' rel='stylesheet' type='text/css'>
    <style type="text/css">
        html {
            color: #222222;
            font-family: "Noto Serif";
        }

        h1 {
            font-size: 16pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
        }

        h2 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h3 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h4 {
            font-size: 13pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h5 {
            font-size: 12pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        b {
            font-weight: bold;
        }

        p {
            font-size: 12pt;
        }

        li {
            font-size: 12pt;
        }

        pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
        }

        code {
            color: #e83e8c;
        }

        code.language-input-format {
            color: #222222;
        }

        .testcase_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-width: 480;
            max-height: 340;
        }

        .error_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #f0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-height: 340;
            background-color: #ffeeee;
        }

        .table tbody>tr>td.vert-aligned {
            vertical-align: middle;
        }
    </style>
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
            tex2jax: {
                inlineMath: [["$", "$"]],
                processEscapes: true,
                processClass: 'input-format|language-input-format'
            }
        });
    </script>
    <script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js?config=TeX-AMS_CHTML">
        </script>
</head>

<body>
    
    <div >
        <h2>Problem A</h2>
<h3><u><strong>A+B</strong></u></h3>
<p>2 つの整数 $A$, $B$ が与えられます。$A + B$ の計算結果を出力してください。</p>
<h3>入力</h3>
<p>入力は以下の形式で標準入力から与えられます。</p>
<pre class="input-format"><code class="language-input-format">$A$ $B$
</code></pre>
<h3>出力</h3>
<p>$A + B$ の計算結果を標準出力に出力してください。</p>
<h3>制約</h3>
<ul>
<li>$1 \leq A, B \leq 10^{9}$</li>
</ul>
<h3>入力例 1</h3>
<pre class="input-format"><code class="language-input-format">3 5
</code></pre>
<h3>出力例 1</h3>
<pre class="input-format"><code class="language-input-format">8
</code></pre>
<h3>入力例 2</h3>
<pre class="input-format"><code class="language-input-format">10000 100
</code></pre>
<h3>出力例 2</h3>
<pre class="input-format"><code class="language-input-format">10100
</code></pre>
    </div>
    
</body>

</html>